Jenkinsfile

```diff
@@ -46,132 +49,107 @@ pipeline {
       steps {
         script {
           // Build site with CV Buildkit
-          sh "civibuild create ${params.CIVIHR_BUILDNAME} --type hr17 --civi-ver 4.7.27 --hr-ver staging --url $WEBURL --admin-pass $ADMIN_PASS"
+          sh "civibuild create ${params.CIVIHR_BUILDNAME} --type drupal-clean --civi-ver 4.7.27 --url $WEBURL --admin-pass $ADMIN_PASS"
+
           sh """
             cd $DRUPAL_MODULES_ROOT/civicrm
             wget -O attachments.patch https://gist.githubusercontent.com/davialexandre/199b3ebb2c69f43c07dde0f51fb02c8b/raw/0f11edad8049c6edddd7f865c801ecba5fa4c052/attachments-4.7.27.patch
             patch -p1 -i attachments.patch
             rm attachments.patch
           """
 
-          // Change git remote of civihr ext to support dev version of Jenkins pipeline
-          changeCivihrGitRemote()
-
-          // Get repos & branch name
+          // Get target and PR branches name
           def prBranch = env.CHANGE_BRANCH
-          def envBranch = env.CHANGE_TARGET
+          def envBranch = env.CHANGE_TARGET ? env.CHANGE_TARGET : env.BRANCH_NAME
```

Comment:
Is this related to the performance improvements or why was it changed?

Reply:
It is. This was necessary because now the civihr code is not downloaded by civibuild anymore. So we need a way to tell which base/enviroment branch to use. This also fix an issue introduced by https://github.com/civicrm/civihr/pull/2372 where `staging` would be used as a base branch to run the tests when something was merged to `master`. I'll add this to the PR description too

Reply:
Thanks, it's a lot clearer now